spec/spec_helper.rb

```diff
@@ -13,6 +13,7 @@
 
 RSpec.configure do |config|
   # see more settings at spec/rails_helper.rb
+  config.disable_monkey_patching!
```

Comment:
more info on this here: https://rubydoc.info/github/rspec/rspec-core/RSpec%2FCore%2FConfiguration:disable_monkey_patching!